samcli/local/lambdafn/zip.py

```diff
@@ -169,7 +170,7 @@ def unzip_from_uri(uri, layer_zip_path, unzip_output_dir, progressbar_label):
         with open(layer_zip_path, "wb") as local_layer_file:
             file_length = int(get_request.headers["Content-length"])
 
-            with progressbar(file_length, progressbar_label) as p_bar:
+            with progressbar(file_length, progressbar_label, get_text_stderr()) as p_bar:
```

Comment:
what is the reasoning for this? should we add a comment, could there be a regression?